sakura_core/env/CShareData_IO.cpp

```diff
@@ -1639,6 +1639,7 @@ void CShareData_IO::ShareData_IO_Type_One( CDataProfile& cProfile, STypeConfig&
 	cProfile.IOProfileData( pszSecName, LTEXT("bIndentCppStringIgnore")		, types.m_bIndentCppStringIgnore );
 	cProfile.IOProfileData( pszSecName, LTEXT("bIndentCppCommentIgnore")	, types.m_bIndentCppCommentIgnore );
 	cProfile.IOProfileData( pszSecName, LTEXT("bIndentCppUndoSep")	, types.m_bIndentCppUndoSep );
+	cProfile.IOProfileData( pszSecName, LTEXT("bDetectIndentationStyleOnFileLoad "), types.m_bDetectIndentationStyleOnFileLoad );
```

Comment:
誤記見つけました。

```suggestion
	cProfile.IOProfileData( pszSecName, LTEXT("bDetectIndentationStyleOnFileLoad"), types.m_bDetectIndentationStyleOnFileLoad );
```

サクラエディタでは、共有メモリ・タイプ別設定の項目を、そのままINIファイル設定項目名にする仕様です。
マージする前に適切な名前に改善できるなら対応したほうがよいと思います。
もっとも、既存の項目にも「常識の範囲で考えておかしな項目名」がたくさんあるので、修正は必須でないのかも知れません（ただ、余分な空白は削って欲しいです。）